icu4c/source/i18n/number_output.cpp

```diff
@@ -47,6 +48,42 @@ void FormattedNumber::getDecimalQuantity(impl::DecimalQuantity& output, UErrorCo
 impl::UFormattedNumberData::~UFormattedNumberData() = default;
 
 
+UPRV_FORMATTED_VALUE_SUBCLASS_AUTO_IMPL(FormattedNumberRange)
+
+#define UPRV_NOARG
+
+UnicodeString FormattedNumberRange::getFirstDecimal(UErrorCode& status) const {
+    UPRV_FORMATTED_VALUE_METHOD_GUARD(ICU_Utility::makeBogusString())
+    return fData->quantity1.toScientificString();
+}
+
+UnicodeString FormattedNumberRange::getSecondDecimal(UErrorCode& status) const {
+    UPRV_FORMATTED_VALUE_METHOD_GUARD(ICU_Utility::makeBogusString())
+    return fData->quantity2.toScientificString();
+}
+
+void FormattedNumberRange::getDecimalNumbers(ByteSink& sink1, ByteSink& sink2, UErrorCode& status) const {
```

Comment:
You will need to rebase again to pick up your recent API addition, right? (After which this here should no longer show up as an addition.)

Reply:
My branch is up-to-date.  You are seeing it as added because I moved the definition of FormattedNumberRange from numrange_fluent.cpp to number_output.cpp.  This function is deleted from the first and added to the second.